### DISCUSSION TITLE
Link Microsoft Bot Framework to matterbridge API

### DISCUSSION THREAD

**Discussion by MRolandd** [42wim/matterbridge#1535] [Help / Q&A]
How can I like the Microsoft's Bot Framework to work with the matterbridge API ?
I want to have a local microsoft bot framework, meaning that I'm not working with Azure

**Comment by 42wim** [comment]
You can find more info about the matterbridge API here: https://github.com/42wim/matterbridge/wiki/API
And also a list of projects: https://github.com/42wim/matterbridge#api that use it.
